client-local/src/main/java/com/cloudera/livy/client/local/LocalConf.java

```diff
@@ -48,6 +48,7 @@
     CLIENT_REPL_MODE("repl", false),
 
     LIVY_JARS("jars", null),
+    SPARKR_PACKAGE("spark.package", null),
```

Comment:
oops, missed this before, but should be `sparkr.package`. I'll fix it during merge.